app/views/comments/_question_comment.html.erb

```diff
@@ -0,0 +1,5 @@
+<%= form_for [@question, @comment] do |f|%>
```

Comment:
Looking at it once more, we could have used one partial for new comments instead of two (see below, pretty sure that would work). But we can always DRY it up later.

app/views/comments/_new.html.erb
<%= form_for [target, comment] do |f|%>
  <%= f.label :comment %>
  <%= f.text_field :text %><br><br>
  <%= f.submit 'Comment!' %>
<% end %>